build.gradle

```diff
@@ -29,6 +29,7 @@ subprojects {
         maven { url "https://adcolony.bintray.com/AdColony" }
         maven { url 'https://dl.bintray.com/ironsource-mobile/android-sdk'}
         maven { url 'https://tapjoy.bintray.com/maven' }
+        maven { url "https://jitpack.io" }
```

Comment:
small suggestion: I know we use jitpack for RC release but MoPub don't use this maven to fetch our aar.